scapy/arch/windows/__init__.py

```diff
@@ -69,16 +71,60 @@ def _encapsulate_admin(cmd):
     # To get admin access, we start a new powershell instance with admin rights, which will execute the command
     return "Start-Process PowerShell -windowstyle hidden -Wait -Verb RunAs -ArgumentList '-command &{%s}'" % cmd
 
+class _PowershellManager(Thread):
```

Comment:
Could you make this kind of changes PEP-08 compliant (line length, docstrings, ...) ?